### PR TITLE
Add a Sec-CH-UA-Form-Factor hint

### LINKED ISSUE #333: Hints to identify different form factors
User Agent strings currently contain Mobile to indicate a hand held mobile device. As we consider form factors like TV, VR head sets, automotive and more it would be very useful to know what form factor the customer is browsing a website on so webapp owners can fine tune the experience.

Eg. a VR experience might want to make things more friendly for hand gestures, automotive larger touch targets

Could we consider a new Client Hint for this purpose?

### PR DESCRIPTION
**Pull request by miketaylr**
Fixes #333

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/ua-client-hints/pull/338.html" title="Last updated on Mar 29, 2023, 8:56 PM UTC (2fbf40b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/ua-client-hints/338/ec059e9...miketaylr:2fbf40b.html" title="Last updated on Mar 29, 2023, 8:56 PM UTC (2fbf40b)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -435,13 +435,16 @@ has defined a number of properties for itself:
 *   <dfn for="user agent" export>brand</dfn> - The [=user agent=]'s commercial name (e.g.,
       "cURL", "Edge", "The World's Best Web Browser"), which MUST be shorter than 32 [=ASCII alpha=]
       characters.
-*   <dfn for="user agent" export>significant version</dfn> - The marketing version which includes
-      distinguishable web-exposed features (e.g., "72", "3", or "12.1"), corresponding to the
-      [=user agent=], or any of the brands in its [=brands=] list (e.g., rendering engine or
-      any other [=equivalence classes=] full version).
+*   <dfn for="user agent" export>form-factor</dfn> - The form-factor of a device, historically
+      represted as a &lt;[=deviceCompat=]&gt; token in the User-Agent string (e.g., "Tablet", "VR",
```
Comment (miketaylr) [inline index.bs:439]: https://compat.spec.whatwg.org/#devicecompat was just exported today, should give it a day or so for other tooling to catch up.

Comment (miketaylr) [inline index.bs:439]: https://compat.spec.whatwg.org/#devicecompat was just exported today, should give it a day or so for other tooling to catch up.

**Comment by miketaylr** [comment]
PTAL @yoavweiss!

**Review by yoavweiss — APPROVED** [review]
LGTM

**Comment by miketaylr** [comment]
thanks!

**Pull request merged**